gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Dialog.java

```diff
@@ -128,8 +128,8 @@ public Table getContentTable () {
 		return contentTable;
 	}
 
-	public Table getButtonTable () {
-		return buttonTable;
+	public Table getButtonsTable () {
```

Comment:
Heh, somehow for me buttonsTable seemed better name for that table. I will change it then.